redaxo/src/addons/be_style/functions/rex_be_style_compile.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Functions for be_style.
+ *
+ * @package redaxo\be_style
+ */
+
+/**
+ * Converts Backend SCSS files to CSS.
+ */
+function rex_be_style_compile()
```

Comment:
Funktionen wollten wir m.E. eher vermeiden und auf Klassen setzen. Ich fände es hier dennoch ok.

Reply:
wenn man wirklich mit ner richtigen klasse und statischer methode arbeitet, bräuchte man das require nicht und könnte auf das autoloading vertrauen.

sorry @bloep , dass ich dich mit "extrahiere es in eine funktion" auf den holzweg geführt habe